apps/ejabberd/src/ejabberd_s2s.hrl

```diff
@@ -0,0 +1,23 @@
+%% These error macros will change the xmpp errors(xml) to binaries
+%% All errors are defined in mongoose_xmpp_errors.erl
+%% Previously they were defined in up to three different modules
+
+-define(INVALID_NS_ERR_TO_BIN,
+    exml:to_binary(mongoose_xmpp_errors:invalid_namespace())).
+
+-define(HOST_UNKNOWN_ERR_TO_BIN,
+    exml:to_binary(mongoose_xmpp_errors:host_unknown())).
+
+-define(INVALID_XML_ERR_TO_BIN,
```

Comment:
Could be a special, separate function in `mongoose_xmpp_errors`, e.g. `xml_not_well_formed_bin()`.